### PR TITLE
add delete data section to working with data

### PR DESCRIPTION
**Pull request by alorenzo175**

### CODE REVIEW

File: documentation/dashboard/working_with_data.md [hunk 1]
```diff
@@ -290,7 +290,18 @@ may download more than one year of data by making multiple requests.
 
 <div class="my-3"></div>
 
-### Create New Report
+Delete Data
+-----------
+{: .anchor}
+
+Individual forecast or observation values cannot currently be deleted from the
```
Comment (wholmgren) [inline documentation/dashboard/working_with_data.md:297]: Start with comment about how users can delete entire time series and metadata, including any requirements about deleting time series data before metadata and what limitations might exist due to data sharing (e.g. you share a site with me, I create an observation/observation values on that site, and you want to delete your site).

File: documentation/dashboard/working_with_data.md [hunk 2]
```diff
@@ -290,7 +290,18 @@ may download more than one year of data by making multiple requests.
...
+Solar Forecast Arbiter. However, one may upload NaN data
+(null in JSON, 'NaN' or an empty field in CSV) at the appropriate
+timestamps as a workaround.
+
+
+Create New Report
+-----------------
```
Comment (wholmgren) [inline documentation/dashboard/working_with_data.md:304]: I think number of `#` is more clear than type of underline, but I don't object.

File: documentation/dashboard/working_with_data.md [hunk 3]
```diff
@@ -290,7 +290,18 @@ may download more than one year of data by making multiple requests.
 
 <div class="my-3"></div>
 
-### Create New Report
+Delete Data
```
Comment (wholmgren) [inline documentation/dashboard/working_with_data.md:293]: needs an entry in `_includes/dashboard_data_sidebar.html`

File: documentation/dashboard/working_with_data.md (hunk 2 above)
Comment (alorenzo175) [reply documentation/dashboard/working_with_data.md:304]: yeah, just stuck to the style elsewhere in this doc

File: documentation/dashboard/working_with_data.md [hunk 4]
```diff
@@ -290,7 +290,26 @@ may download more than one year of data by making multiple requests.
...
+all values for that object are also deleted automatically.
+Individual forecast or observation values cannot currently be deleted from the
+Solar Forecast Arbiter. However, one may upload NaN data
+(null in JSON, 'NaN' or an empty field in CSV) at the appropriate
+timestamps as a workaround.
+Site objects may only be deleted after all observations at said site
+are also deleted.
+In the case of a site that has been shared with users in other organizations,
+contact the users to delete those observations referencing the site or contact
+the Solar Forecast Arbiter administrators.
```
Comment (wholmgren) [inline documentation/dashboard/working_with_data.md:308]:
```suggestion
the Solar Forecast Arbiter administrators at [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org).
```

**Review by wholmgren — APPROVED** [review]

**Pull request merged**